pkg/providers/providers.go

```diff
@@ -41,9 +47,5 @@ func New(u string) (Provider, error) {
 		return newGitHub(purl)
 	}
 
-	if strings.Contains(purl.Host, "hub.docker.com") || strings.Contains(purl.Host, "docker.io") {
```

Comment:
Before this I could do `bin install github.com/franela/matipan`. Can we fallback to `https://` seems like this won't work anymore now. Can we add a case for that there?